app/proguard-rules.pro

```diff
@@ -18,6 +18,7 @@
 -keepattributes SourceFile,LineNumberTable
 -dontobfuscate
 
+-keep class com.jcraft.jsch.**
```

Comment:
I was getting errors in the log about not being able to resolve the Random class from jsch, not sure if that's mission-critical so feel free to drop this if it's not going to break things.

Reply:
Well, whatever. Let's leave it in and stop worrying about Jsch alltogether.